src/python/WMCore/Database/MySQLCore.py

```diff
@@ -6,37 +6,30 @@
 """
 
 import copy
+import sys
```

Comment:
Probably a leftover when you were working on it? This import is no longer needed.

Reply:
Oh yes, you are correct! Thanks!